test/fuzz/tcti/tss2-tcti-fuzzing.map

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need a map file for a noinst library ?
This you can just export all symbols, who cares ?